linux_os/guide/system/software/integrity/software-integrity/aide/var_aide_scan_notification_email.var

```diff
@@ -0,0 +1,16 @@
+documentation_complete: true
+
+title: Integrity Scan Notification Email Address
+
+description: |-
+    Specify the email address for designated personnel if baseline
```

Comment:
"for notifications to designated personnel" perhaps?

Reply:
Fixed, thanks.